src/staticOptions.ts

```diff
@@ -31,6 +31,19 @@ const staticOptions = asLanguageClientOptionsById({
       { scheme: 'file', language: 'c' },
       { scheme: 'file', language: 'cpp' }
     ],
+    middleware: {
```

Comment:
Is this that the objective-c language has the same commented line:
```ts
// vscodeExtensionIds: ['cpptools']
```
Should this middleware be added to objective-c as well ?

Reply:
well spotted! fixed and amended